### ISSUE TITLE
Handlebars inline formatting

### ISSUE THREAD

**Issue by m-nagy** [beautify-web/js-beautify#1173]
# Description
Handlebars inline tags not formatted probably

# Input
The code looked like this before beautification:
```
{{title "abc"}}
{{log "abc"}}
```

# Expected Output
The code should have looked like this after beautification:
```
{{title "abc"}}
{{log "abc"}}
```

# Actual Output
The  code actually looked like this after beautification:
```
{{title "abc"}}{{log "abc"}}
```

# Steps to Reproduce
When writing many handlebars tags inline like the following
{{title "abc"}}{{log "abc"}}
VSCode formatter (Alt+Shift+F) adds all inline tags in the same line and no way to configure the editor to add each tag on a single line which is very annoying

## Environment
OS: Windows_NT ia32 6.1.7601

## Settings
Example:
```json
{
    "html.format.indentHandlebars": true,
    "editor.formatOnSave": true,
    "html.format.endWithNewline": true,
}
```

**Comment by v3ss0n** [comment]
We need handlebars support !!

**Comment by bitwiseman** [comment]
#660

**Comment by jakeleboeuf** [comment]
Yes please!!

**Comment by ifahrentholz** [comment]
Pls...

**Comment by MrChriZ** [comment]
+1

**Comment by m-diiorio** [comment]
+1 Please add an option to avoid inline formatting of tags. This is a problem also with handlebars comments

**Comment by MacKLess** [comment]
This has been fixed as of v1.8.1
